endpointmanager/pkg/chplendpointquerier/bizmaticsparser.go

```diff
@@ -0,0 +1,24 @@
+package chplendpointquerier
+
+import (
+	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/helpers"
+	log "github.com/sirupsen/logrus"
+)
+
+func BizmaticsBundleParser(CHPLURL string, fileToWriteTo string) {
```

Comment:
this function is *so close* to being the same as `cernerquerier.go` that we could pull out the few extra lines in the cerner file to be in the chplendpointquerier and then have both of those endpoint lists call the same function? Do you think it's worth doing that?